br-webui/test/test-query-udevadm.js

```diff
@@ -0,0 +1,28 @@
+var express = require('express');
+var app = express();
+var path = require("path");
+const child_process = require('child_process');
+
+var COMPANION_DIR = "../.."
+
+var server = app.listen(2770, function() {
+    var host = server.address().address;
+    var port = server.address().port;
+    console.log("App running at http://%s:%s", host, port);
+});
+
+app.get('/', function(req, res) {
+    res.sendFile(path.join(__dirname + "/test-query-udevadm.html"));
+});
+
+app.get("/udevadm", function(req, res) {
+    var pattern = "";
+    if (req.query.pattern) {
+        pattern = "--pattern=" + req.query.pattern;
+    }
+    console.log("got request for query:" + req.query.pattern);
+    
+    var result = child_process.execSync("python3 " + COMPANION_DIR + "/tools/query-udevadm.py --indent=2 " + pattern);
```

Comment:
should we use python3 -u ?

Reply:
I don't think it will make a difference to use unbuffered output.

Reply:
Why we do it with python and not with python3 ?

Reply:
where? Perhaps we did it for no reason or some lack of understanding of the option.

Reply:
https://github.com/bluerobotics/companion/pull/212/files#diff-bf78ead9729f5655d7c00aecf5c6df75R77

Reply:
Yeah, I don't know the reason for it here. Except that these scripts update the output in webui asynchronously, so maybe it is needed for that. I think that -u is not needed here (this PR works).